bot/src/notify.ts

```diff
@@ -1,9 +1,69 @@
 import type { AuctionInitialInfo } from 'auctions-core/src/types';
+import BigNumber from 'auctions-core/node_modules/bignumber.js/bignumber';
 import { sendNotification } from './twitter';
 
+const DECIMAL_PLACES_DEFAULT = 2;
+const DECIMAL_PLACES_MAX = 5;
+
+function isValidNumber(value: BigNumber): boolean {
```

Comment:
Since you (hopefully) copied this from the frontend code: Can it be shared between the two? For example it can be a `formatToAutomaticDecimalPoints` function that lives inside `core/src/helpers` and is used both here and in the frontend component?